src/panels/iframe/ha-panel-iframe.ts

```diff
@@ -11,7 +11,7 @@ class HaPanelIframe extends LitElement {
 
   @property({ type: Boolean }) narrow!: boolean;
 
-  @property() panel!: PanelInfo<{ url: string }>;
+  @property() panel!: PanelInfo<{ url: string, allow: string }>;
```

Comment:
Not sure:

```suggestion
  @property() panel!: PanelInfo<{ url: string, allow?: string }>;
```

Should check on other optional properties.